nucleus/deploy/model_endpoint.py

```diff
@@ -7,6 +7,22 @@
 TASK_FAILURE_STATE = "FAILURE"
 
 
+class SyncModelEndpoint:
+    def __init__(self, endpoint_id: str, client):
+        self.endpoint_id = endpoint_id
+        self.client = client
+
+    def __str__(self):
+        return f"SyncModelEndpoint <endpoint_id:{self.endpoint_id}>"
+
+    def predict_one(self, url):
```

Comment:
Can we just call this `predict`?

Also, non-url inputs are in a different ticket, right?

Reply:
re non-url inputs, they are in a different ticket, and I'm inclined to just open a different pr for them as well at this point

re `predict`, have offline discussion